bin/lolcommits

```diff
@@ -116,7 +87,9 @@ def do_capture
     :config          => configuration
   }
 
-  fork_me? do
```

Comment:
perhaps whether we fork or not should be moved into the configuration object, and then that should be referenced inside the new `ProcessRunner` rather than passing it to this `fork_me?(please)` method ..